### PR TITLE
[翻译完毕] type

### PR DESCRIPTION
**Pull request by rexwangcc**
## Open questions: 
- `numerical literals` was used in the original text while it's probably supposed to be `numeric literals`.
- Not sure if `literals` -> `字面值` is a good translation.
- Some unrelated `.po` files are also touched and checked out, which makes the PR a bit noisy.

### CODE REVIEW

**Comment by archibate** [comment]
抱歉！会有一些无关.po文件是因为taichi更新了英文文档。
@StephenArk30 或许我们希望由maintainer来保持文档最新，而不是contributor的PR？(通过不定时`make merge`)

File: locales/zh_CN/LC_MESSAGES/type.po [hunk 1]
```diff
@@ -214,4 +220,16 @@ msgid ""
 "mat_int = mat.cast(int)\n"
 "mat_int2 = mat.cast(ti.i32)"
 msgstr ""
-
+"a = 1.4\n"
+"b = ti.cast(a, ti.i32)\n"
+"c = ti.cast(b, ti.f32)\n"
+"\n"
+"# 同样地, 使用 ``int()`` 和 ``float()``\n"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/type.po]: 建议改成: 或者你也可以/等价地

File: locales/zh_CN/LC_MESSAGES/type.po [hunk 2]
```diff
@@ -172,14 +175,17 @@ msgstr ""
...
-msgstr ""
+msgstr "默认精度"
 
 #: ../../type.rst:63
 msgid ""
 "By default, numerical literals have 32-bit precisions. For example, "
 "``42`` has type ``ti.i32`` and ``3.14`` has type ``ti.f32``. Default "
 "precisions can be specified when initializing Taichi:"
 msgstr ""
+"默认情况下，数值字面值（numeric literals）的精度均为32位（32-bit）。例"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/type.po]: 让我们把literal翻译成“字面常量”吧。

**Review by archibate — COMMENTED** [review]
感谢你的耐心和翻译，这对我们很有帮助！LGTM except for some little nits.

File: locales/zh_CN/LC_MESSAGES/type.po (hunk 2 above)
Comment (rexwangcc) [reply locales/zh_CN/LC_MESSAGES/type.po]: 好的，已修改。

File: locales/zh_CN/LC_MESSAGES/type.po (hunk 1 above)
Comment (rexwangcc) [reply locales/zh_CN/LC_MESSAGES/type.po]: 好的，已修改~

**Comment by rexwangcc** [comment]
@archibate 十分感谢PR review，已按照建议进行了更新，烦请过目。

**Review by archibate — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**